<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit_window</class>
 <widget class="QMainWindow" name="Deposit_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:10px;
background-color: rgb(66, 66, 66)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Сумма вклада:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Процентная ставка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Налоговая ставка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Срок размещения (мес):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Капитализация процентов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Снятия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Периодичность выплат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сумма начислений:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>345</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Сумма вклада с процентами:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Удержано налогов:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accrued_interest">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tax">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_res">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(98, 160, 234);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_summ">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_Button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(98, 160, 234);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Отчистить </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>228</y>
      <width>831</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>831</width>
      <height>151</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>831</width>
      <height>151</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);
border-radius:10px;
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_interest_rate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_n">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_tax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_periodicity_payments">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>В конце срока</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_periodicity_payments_on">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_add">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пополнение раз в месяц</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_add">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>130</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_sub">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Снятие раз в месяц</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_sub">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_capitalization">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Капитализация раз в месяц</string>
    </property>
   </widget>
   <zorder>label_12</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_accrued_interest</zorder>
   <zorder>label_res</zorder>
   <zorder>label_tax</zorder>
   <zorder>pushButton_res</zorder>
   <zorder>line_summ</zorder>
   <zorder>clear_Button</zorder>
   <zorder>line_interest_rate</zorder>
   <zorder>line_n</zorder>
   <zorder>line_tax</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>radioButton_periodicity_payments</zorder>
   <zorder>radioButton_periodicity_payments_on</zorder>
   <zorder>checkBox_add</zorder>
   <zorder>line_add</zorder>
   <zorder>checkBox_sub</zorder>
   <zorder>line_sub</zorder>
   <zorder>checkBox_capitalization</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
